### PR TITLE
docs: Update Telemetry Doc

### PR DESCRIPTION
**Pull request by jfermi**
Updated telemetry.md and corrected broken image links in tests-creating.md.

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: docs/telemetry.md [hunk 1]
```diff
@@ -25,3 +29,25 @@ The collected data looks like this.
   "type": "track"
 }
 ```
+
+## **What We Collect**
+
+Analytics tracked include:
+- The unique id generated from the MAC address.
+- The testkube version.
+- The command being executed.
```
Comment (exu) [inline docs/telemetry.md:38]: Maybe we could point out that we don't track any custom parameters and flags passed to a command by user - only top level testkube commands for general stats ?

Comment (exu) [reply docs/telemetry.md:38]: @jfermi ^^

Comment (nicufk) [reply docs/telemetry.md:38]: @exu, in the details field  we are tracking the whole command

Comment (jfermi) [reply docs/telemetry.md:38]: So it sounds like we can't say "we don't track customer parameters and flags passed" right now?

File: docs/tests-creating.md [hunk 2]
```diff
@@ -21,15 +21,15 @@ To create your first Postman collection in Testkube, export your collection into
 
 Right click on your collection name:
 
-![create postman collection step 1](img/test-create-1.png)
+![create postman collection step 1](img/script-create-1.png)
```
Comment (vsukhin) [inline docs/tests-creating.md:24]: We don't have `script` anymore in our terms. Either it should be new test-something image or rename exsiting image, if it's already with tests

Comment (jfermi) [reply docs/tests-creating.md:24]: I can rename the image file.  I left them the same in case they were used by something I was not aware of.

Comment (jfermi) [reply docs/tests-creating.md]: I changed the image file names to test-create... and updated tests-creating.md.

**Review by vsukhin — APPROVED** [review]

**Pull request merged**